packages/terra-embedded-component/src/EmbeddedComponent.jsx

```diff
@@ -1,31 +1,135 @@
+/* global SystemJS */
 import React from 'react';
 import PropTypes from 'prop-types';
-import classNames from 'classnames/bind';
-import 'terra-base/lib/baseStyles';
-import styles from './EmbeddedComponent.scss';
+import axios from 'axios';
+import 'systemjs/dist/system-production';
+import { loadStylesheet } from './assets';
 
-const cx = classNames.bind(styles);
+// Expose AMD required globals on the window
+window.define = window.SystemJS.amdDefine;
+window.require = window.requirejs = SystemJS.amdRequire;
 
 const propTypes = {
- /*
- * Content to be displayed as the name
- */
-  name: PropTypes.string,
+
+  /**
+   * The basePath to load the manifest and manifest asset dependencies from.
+   * */
+  basePath: PropTypes.string.isRequired,
+
+  /**
+   * A placeholder to display while the component is loading.
+   * */
+  placeholder: PropTypes.element,
+
+  /**
+   * The number of milliseconds to wait before timing out.
```

Comment:
add documentation that the component will throw an error if the timeout is exceeded and that it should be handled in the parent component.

Reply:
Updated docs for timeout error handling: https://github.com/cerner/terra-framework/tree/terra-embedded-component/packages/terra-embedded-component/docs